ammico/cropposts.py

```diff
@@ -326,12 +326,14 @@ def crop_media_posts(
     # get the reference images with regions that signify areas to crop
     ref_views = []
     for ref_file in ref_files:
-        ref_view = cv2.imread(ref_file)
+        ref_file_path = ref_files[ref_file]["filename"]
+        ref_view = cv2.imread(ref_file_path)
         ref_views.append(ref_view)
     # parse through the social media posts to be cropped
     for crop_file in files:
-        view = cv2.imread(crop_file)
-        print("Doing file {}".format(crop_file))
+        crop_file_path = files[crop_file]["filename"]

```

Comment:
same here, `crop_file["filename"]` should be sufficient

Reply:
Oh..., yeah, sure 🤦.